stacklight_tests/tests/test_alertmanager.py

```diff
@@ -0,0 +1,23 @@
+import socket
+
+
+class TestAlertmanager(object):
+
+    def test_alertmanager_endpoint_availability(self, cluster, prometheus_config):
+        """Check that alertmanager endpoint is available.
+
+        Scenario:
+            1. Get alertmanager endpoint
+            2. Check that alertmanager endpoint is available
+        Duration 1m
+        """
+        port = int(prometheus_config["prometheus_alertmanager"])
+        cfg = [host for host in cluster.hosts
+               if host.fqdn.startswith("cfg")][0].address
+        try:
+            s = socket.socket()
+            s.connect((cfg, port))
+            s.close()
+        except socket.error:
+            return False
```

Comment:
What test result will be in this case, I suppose test will be marked as passed, so there need to rewrite this part of code as:
result = True
try:
    ...
except socket.error: 
    result = False
assert result